core-java-modules/core-java/src/main/java/com/baeldung/iterationcounter/ForLoopIterationCounter.java

```diff
@@ -0,0 +1,49 @@
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.ListIterator;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class ForLoopIterationCounter {
+    private final List<String> colors = Arrays.asList("red", "blue", "yellow", "green");
```

Comment:
This probably ought to be `private static final` and have an uppercase name - e.g. `COLORS`